src/client/client.ts

```diff
@@ -354,10 +367,20 @@ export class Client {
    * ```
    */
   public async events(req?: EventsRequest): Promise<GetEventsResponse> {
-    const query: Record<string, number> = {};
+    const query: Record<string, string> = {};
     if (req?.cursor && req.cursor > 0) {
-      query.cursor = req.cursor;
+      query.cursor = req.cursor.toString();
     }
+
+    Object.entries(req?.filter ?? {}).forEach(([key, value]) => {
```

Comment:
Can we turn this into this:

```
    for (const [key, value] of Object.entries(req?.filter ?? {})) {
      if (typeof value === "number" && value < 0) {
        continue;
      }
      if (typeof value !== "undefined") {
        query[key] = value.toString();
      }
    }
```
Biome is yelling about `forEach` use. And I feel like this version is easier to read.

Reply:
Done! Weird I didn't get the warning.